services/core-api/app/api/utils/setup_marshmallow.py

```diff
@@ -106,7 +108,7 @@ class Meta(object):
                     mapper = inspect(class_)
                     for k, v in class_._ModelSchema.__dict__.items():
                         if type(v) == FieldTemplate:
-                            current_app.logger.debug(f'creating field for {k} on {class_}')
+                            #current_app.logger.debug(f'creating field for {k} on {class_}')
```

Comment:
does this need to be here?

Reply:
yes i think they should stay, reactivated those lines

Reply:
They're pretty noisy on startup.